### DISCUSSION TITLE
can't run forge init  without getting an error

### DISCUSSION THREAD

**Discussion by solidity-cobra** [Cyfrin/foundry-full-course-cu#1521] [General]
i keep getting this error
 "Cannot run `init` on a non-empty directory.
Run with the `--force` flag to initialize regardless."

and my directory is empty.

**Comment by EngrPips** [comment]
This seem weird, are you using `wsl` ?

**Reply by solidity-cobra** [reply]
yes sir

**Reply by solidity-cobra** [reply]
@EngrPips i was able to figure it out, i had to delete the folder i created then run forge init in a new folder in ubuntu.
thanks sir

**Reply by EngrPips** [reply]
Honestly i am really happy that you figure it out. keep crushing them

**Comment by solidity-cobra** [comment]
@EngrPips  please i need your help
i can't figure out what i did wrong
![Screenshot (1)](https://github.com/Cyfrin/foundry-full-course-f23/assets/161164747/0e2f58ca-9200-4a40-b537-a2d73d40406e)

**Reply by EngrPips** [reply]
I have never used `wsl` so issues related to them are really hard for me to grasp but i would say open a folder and try run `forge init` in that new folder.

**Reply by solidity-cobra** [reply]
ohh sorry i sent the wrong screenshot
please check this out......its my code
![Screenshot (2)](https://github.com/Cyfrin/foundry-full-course-f23/assets/161164747/125fe4c6-8202-468d-9997-0403bfad4e24)

**Reply by EngrPips** [reply]
I think to run a script the command is suppose to be 

```javascript
forge script script/DeploySimpleStorage.s.sol:DeploySimpleStorage
```
and why do you have a space between the `vm.startBroadcast` and `()` ? , or am i not seeing correctly ?

**Reply by solidity-cobra** [reply]
alright thanks, i will try it out

**Reply by EngrPips** [reply]
Alright, looking forward to see how that plays out.

**Reply by solidity-cobra** [reply]
it still says start broadcast not found

**Reply by EngrPips** [reply]
Maybe push your code to github and share me the link to the repo, so that i can set it up locally on my machine and try debug it.

**Comment by ARomano79** [comment]
> and why do you have a space between the `vm.startBroadcast` and `()` ? , or am i not seeing correctly ?
 I would check this out

**Reply by solidity-cobra** [reply]
there's actually space but that's not the cause of the error

**Comment by solidity-cobra** [comment]
@EngrPips  sorry for the late reply sir, i've been occupied with school. i will reach out when i am free sir.
thanks for helping me.

**Reply by EngrPips** [reply]
No problem at all friend, take your time
